modules/nf-core/miranda/main.nf

```diff
@@ -0,0 +1,49 @@
+process MIRANDA {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::miranda=3.3a" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/miranda:3.3a--h779adbc_3':
+        'quay.io/biocontainers/miranda:3.3a--h779adbc_3' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    path(mirbase)
+
+    output:
+    tuple val(meta), path("*.txt"), emit: txt
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    miranda \\
+        $mirbase \\
+        $fasta \\
+        $args \\
+        -out ${prefix}.out
```

Comment:
Shouldn't this be ${prefix}.txt?

Reply:
the output from `miRanda` is a mess and needs to be reformatted, so this `.out` file is temporary. 

The code below it is present in virtually all `miRanda` workflows, definitely worthwhile including it in the module :)